lib/models/device.js

```diff
@@ -893,7 +906,7 @@ const getDeviceModel = function (deps, opts) {
 						},
 					}),
 				)
-				.return(undefined),
+				.return(),
```

Comment:
How about converting this one as well?

Reply:
I specifically left this because it's an easy pattern to find/remove in an async/await world whereas in a promise world it's quite a pain, so leaving it as is makes life easier overall imo